corehq/apps/builds/README.rst

```diff
@@ -107,7 +107,6 @@ Adding CommCare (J2ME) Builds to CommCare HQ
 - In the second section `Import a new build from the build server`
 
    #. In the Version field input the version in `x.y.z` format
-   #. In `Build Number` input any number (it doesn't matter what number)
```

Comment:
Due to the Build Number being arbitrary I made it optional for the user and will populate it with random number in corehq/apps/builds/views.py if not specified.